docker/README.md

```diff
@@ -0,0 +1,144 @@
+$ docker attach $(sudo docker ps --filter "name=dgen" --format "{{.ID}}") # If dgen_1 is not found
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ python dgen_model.py # Run scenario
+(dg3n) dgen@cc6e2e5f70b5:/opt/dgen_os/python$ exit # to exit
+$ docker-compose up -d # If you exit, you have to re-up the container if you want to re-attach
+```
+
+### Troubleshooting common issues
+
+#### psycopg2.OperationalError: connection to server
+
+Wait 5-10 minutes for the postgres database to finish starting.
+
+#### General errors and issues
+
+Try clearing your `~/dgen_data` and starting over. Make sure to provide time for the datasets to fully download on the re-attempt.
+
+```bash
+$ docker-compose down
+$ rm -f ~/dgen_data/*
+$ docker system prune -a
+$ docker volume prune -f
+```
+
+### Disabling auto-start for the dgen virtual environment
+
+By default, logging into the `dgen` container automatically activates the `dgen` virtual environment.  For ease of use, its recommended to leave this the default.
+
+To disable this behavior, edit the `docker-compose.yml` file in this directory and set the following environment variable:
+
+```yaml
+services:
+  dgen:
+    environment:
+      DGEN_DISABLE_AUTO_START: 1
+```
+
+### Customizing the dataset
+
+By default, the dgen container uses the [Deleware residential dataset](https://oedi-data-lake.s3.amazonaws.com/dgen/de_final_db/dgen_db.sql).
+
+You can find more datasets using the links below:
+- [DGen Dataset Submissions on OpenEI](https://data.openei.org/submissions/1931)
+- [DGen Dataset S3 Viewer](https://data.openei.org/s3_viewer?bucket=oedi-data-lake&prefix=dgen%2F)
+
+You can customize the dataset used by overriding the DGEN_DATAFILE_URL and DGEN_AGENTFILE_URL variables in `docker-compose.yml` and then editing `~/dgen_data/input_sheet_final.xlsm` using Excel.
+
+Below will walk through the process of using the [Colorado residential dataset](https://oedi-data-lake.s3.amazonaws.com/dgen/co_final_db/dgen_db.sql).
+
+Update the `docker-compose.yml` to use the co_final_db sql download and set the variable to force remove the database.  This will result in dataloss from previous runs, if this is a concern, please make backups before proceeding with the below steps.
+
+```yaml
+services:
+  postgis:
+    environment:
+      DGEN_DATAFILE_URL: https://oedi-data-lake.s3.amazonaws.com/dgen/co_final_db/dgen_db.sql
+      DGEN_AGENTFILE_URL: https://oedi-data-lake.s3.amazonaws.com/dgen/co_final_db/agent_df_base_res_co_revised.pkl
+      DGEN_FORCE_DELETE_DATABASE: 1 # Clear all the data in the database to reload the Colorado dataset, Warning this will remove your existing data.
+```
+
+Edit the excel document `~/dgen_data/input_sheet_final.xlsm` using Excel (Enable macros), edit the Region to Analyize to `Colorado` and Markets to `Only Residential`, then click Save Scenario.
+
+Restart your containers with the above options.  This will remove all your existing data and download the new Colorado dataset.
```

Comment:
```suggestion
Next, restart your containers using the above options.  This will remove all your existing data and setup your Docker Container to use the Colorado files instead.
```